efel/io.py

```diff
@@ -207,3 +209,23 @@ def load_neo_file(file_name: str, stim_start=None, stim_end=None, **kwargs) -> l
         efel_blocks.append(efel_segments)
 
     return efel_blocks
+
+
+def save_feature_to_json(feature_values, filename):
+    """Save feature values as a JSON file."""
+    with open(filename, 'w') as f:
+        json.dump(feature_values, f, default=lambda x: int(x)
+                  if isinstance(x, np.int64) else x)
```

Comment:
Are we sure that we only encounter the np.int64 case? If so, this is ok. If not, you can have a look at how we dealt with that in EModelRunner if you need some inspiration: https://github.com/BlueBrain/EModelRunner/blob/main/emodelrunner/json_utilities.py

Reply:
You are right, we need to consider additional cases too. Thanks for the example, I have implemented something similar